### PR TITLE
frontends.jax.numpy: add support for floor_divide

### LINKED ISSUE #9510: floor_divide

### PR DESCRIPTION
**Pull request by hmahmood24**
Close #9510

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py [hunk 1]
```diff
@@ -2092,3 +2092,43 @@ def test_jax_numpy_fix(
...
+
+
+# floor_divide
+@handle_frontend_test(
+    fn_tree="jax.numpy.floor_divide",
+    dtype_values=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=2,
+        shared_dtype=True,
+        min_value=2.0,
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py:2104]: We need to test against negative values also this limitation should be removed. Yet you can filter values close to zero to avoid zero division error.

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @hmahmood24 
The implementation is correct😊
Below are the changes needed to merge this PR.
Thanks🙂

**Comment by hmahmood24** [comment]
Hi @hirwa-nshuti,

I have made the changes requested and verify that the tests run fine against all backends. Results attached:

image

image

**Review by hirwa-nshuti — APPROVED** [review]
Looks good😊
Going to merge
